server/services/voip/service.ts

```diff
@@ -84,7 +85,34 @@ export class VoipService extends ServiceClass implements IVoipService {
 	}
 
 	async getServerConfigData(type: ServerType): Promise<IVoipServerConfig | null> {
-		return this.VoipServerConfiguration.findOne({ type, active: true });
+		// TODO: Decide the approach we will take regarding settings after the MVP,
+		// For now this work around should be enough.
+
+		// const config = this.VoipServerConfiguration.findOne({ type, active: true });
+
+		const management = type === ServerType.MANAGEMENT;
+		return {
+			_id: '',
+			type,
+			host: settings.get(management ? 'VoIP_Management_Server_Host' : 'VoIP_Server_Host'),
+			name: settings.get(management ? 'VoIP_Management_Server_Name' : 'VoIP_Server_Name'),
+			active: true,
+			_updatedAt: new Date(),
+			...(management
+				? {
+						configData: {
+							port: parseInt(settings.get('VoIP_Management_Server_Port')),
+							username: settings.get('VoIP_Management_Server_Username'),
+							password: settings.get('VoIP_Management_Server_Password'),
+						},
+				  }
+				: {
+						configData: {
+							websocketPort: parseInt(settings.get('VoIP_Server_Websocket_Port')),
+							websocketPath: settings.get('VoIP_Server_Websocket_Path'),
+						},
+				  }),
+		} as IVoipServerConfig;
```

Comment:
looks like the type casting is not needed.
```suggestion
		};
```